packages/sampler-aws-xray/package.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "@opentelemetry/sampler-aws-xray",
-  "version": "0.34.0",
-  "private": true,
-  "description": "OpenTelemetry remote sampler for AWS X-Ray",
+  "version": "0.0.1",
```

Comment:
Total nit: Perhaps 0.1.0 for the initial version? At elast I think that's what other packages have used.  Though I don't know of a real breakage possibly caused by using 0.0.1.